app/config.js

```diff
@@ -39,7 +39,7 @@ define(function (require) {
 		TOTAL_LENGTH: items.length,
 		AMOUNT_TUTORIAL_QUESTIONS: 3,
```

Comment:
Consider distinguishing different objects with constants in this file, e..g one object for database constants, one for session constants something like that.